app/controllers/test/components/mentoring/mentor_inboxes/conversations_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
         trackTitle: "Ruby",
         trackIconUrl: "https://assets.exercism.io/tracks/ruby-hex-white.png",
         menteeAvatarUrl: "https://robohash.org/exercism",
-        menteeHandle: "mentee",
+        menteeHandle: "Mentee",
```

Comment:
I've done this just to have the sorting make sense without having to do case-insensitive sorting :D

Reply:
Ha. Yes!